examples/built-in-string-subtype/built_in_string_subtype.md

```diff
@@ -0,0 +1,11 @@
+# Built-in string subtype
+
+The `ballerina/lang.string` lang library defines the `Char` type, which is a built-in subtype of `string`. A string belongs to the `string:Char` type if it has length 1.
```

Comment:
Cant we add a link here?

Reply:
Does it add any value though? https://lib.ballerina.io/ballerina/lang.string/0.0.0/types#Char is what we have.

Reply:
To the string module. https://lib.ballerina.io/ballerina/lang.string/0.0.0 . Where we mention it in the first sentence. We may have to consistent across BBEs

Reply:
This was considered previously also, but we didn't go ahead with it since it wasn't possible for most BBEs. Unlike with stdlib BBEs lang BBEs don't always correspond to a langlib or they may have multiple relevant langlibs. There's also the possibility of users not knowing the concept of langlibs when they get to certain BBEs.

Anyway since this BBE is about `string:Char` don't think adding a link to the string langlib here is the best thing. Would be more relevant if we did it for the string BBE, but that's one of the first BBEs and it's possible that the user does not know of langlibs when they get there, if they are following the BBEs in order.